.circleci/config.yml

```diff
@@ -18,6 +18,7 @@ templates:
       - image: datadog/datadog-agent-runner-circle:go1199
         environment:
           USE_SYSTEM_LIBS: "1"
+    ressource_class: medium+
```

Comment:
The correct name for that parameter is `resource_class`. Moreover, I'd only change the resource class for the specific jobs that need it (ie. the ones for which it prevents failure, or provides meaningful speed increases), not all of them by default, as this would cost us more money.